lib/ansible/modules/cloud/vmware/vmware_guest.py

```diff
@@ -208,11 +208,10 @@
           will be disconnected but present.'
     - ' - C(iso_path) (string): The datastore path to the ISO file to use, in the form of C([datastore1] path/to/file.iso).
           Required if type is set C(iso).'
-    - ' - C(controller_type) (string): Default value is C(ide). Only C(ide) controller type for CD-ROM is supported for
-          now, will add SATA controller type in the future.'
-    - ' - C(controller_number) (int): For C(ide) controller, valid value is 0 or 1.'
-    - ' - C(unit_number) (int): For CD-ROM device attach to C(ide) controller, valid value is 0 or 1.
-          C(controller_number) and C(unit_number) are mandatory attributes.'
+    - ' - C(controller_type) (string): Valid options are C(ide), C(sata). Default value is C(ide).'
```

Comment:
default is c(sata) not c(ide)?

Reply:
Yeah, if 'controller_type' is not specified then default is 'ide'.

Reply:
I'm not sure from which ESXi version if default CDROM attached to SATA controller for every Guest OS types when deploy manually. Do you know by default is SATA for all guests now? Thanks.